rpc/sigpipe.go

```diff
@@ -0,0 +1,9 @@
+// +build !darwin
+
+package rpc
+
+import "net"
+
+func DisableSigPipe(c net.Conn) error {
+	return nil
```

Comment:
How about I just add a better comment on `DisableSigPipe` that explains it is only available on Darwin?